### DISCUSSION TITLE
How to limit the number of jobs running at once?

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#18326] [Q&A] [answered]
Hello everyone, how do I ensure that a sensor( or a scheduled job) only allows one run at one time?
I am trying to avoid 2 of the same job stepping on each other.

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1701183917227999?thread_ts=1701183917.227999&cid=C01U954MEER).*

**Comment by smackesey** [comment, marked as answer]
See here: https://github.com/dagster-io/dagster/discussions/8414
